airflow/dags/transform_warehouse_full_refresh_sunday/dbt_run_and_upload_artifacts_full_refresh_exclude_rt.yml

```diff
@@ -0,0 +1,77 @@
+operator: 'operators.PodOperator'
+name: 'dbt-run-and-upload-artifacts'
```

Comment:
**suggestion:** I'm not sure whether it's necessary, but it seems that every other `PodOperator` in the repo has the `name` attribute match the file name. Should probably update for the tasks in this DAG.